contrib/oksh/template.py

```diff
@@ -0,0 +1,28 @@
+pkgname = "oksh"
+pkgver = "7.4"
+pkgrel = 0
+build_style = "configure"
```

Comment:
gnu_configure (+ `configure_gen = []`) for this one (it technically isn't, but...)